test/visual/mpl/circuit/test_circuit_matplotlib_drawer.py

```diff
@@ -1117,7 +1117,9 @@ def test_idle_wires_barrier(self):
         circuit.barrier()
 
         fname = "idle_wires_barrier.png"
-        self.circuit_drawer(circuit, output="mpl", cregbundle=False, filename=fname)
+        self.circuit_drawer(
+            circuit, output="mpl", cregbundle=False, filename=fname, idle_wires=False
```

Comment:
I accidentally found that a test with the description "Test that idle_wires False works with barrier" was not doing `idle_wires=False`